realalg/algebraic.py

```diff
@@ -6,13 +6,21 @@
 from math import log10 as log
 from numbers import Integral
 
-import cypari as cp
 import sympy as sp
+try:
```

Comment:
Independently of the installation procedure, this `try/except` in the code is an improvement!